src/app/shell/main/result/hobby-groups-list/hobby-groups-list.component.html

```diff
@@ -1 +1,22 @@
-<p>hobby-groups-list works!</p>
+<div class="cards" fxLayout="row wrap" fxLayoutGap='2%' fxLayoutAlign="flex-start" >
+  <div class="cards_wrapper" *ngFor='let card of cards'>
```

Comment:
it is needed to be moved to card component and used as a web component which encapsulates card logic.

e g:

`
<card-list>
  <ng-container *ngFor="let card of (cards | async)">
    <card [card]="card"></card>
  </ng-container>
</card-list>`

Reply:
done